build/webpack/_production.js

```diff
@@ -16,17 +16,37 @@ export default (webpackConfig) => {
   }
 
   debug('Apply ExtractTextPlugin to CSS loaders.')
+  let appliedExtractCSS = false
   webpackConfig.module.loaders = webpackConfig.module.loaders.map(loader => {
-    if (/css/.test(loader.test)) {
-      const [first, ...rest] = loader.loaders
+    if (!loader.loaders) {
+      return loader
+    }
+
+    let hasCSSLoader = false
+    loader.loaders.forEach(styleLoader => {
```

Comment:
I personally am not fond of the inline function in the if statement, but if thats what you want? This could produce false positives with any loader with css in the name like `postcss` right?